Makefile

```diff
@@ -0,0 +1,55 @@
+TEST?=./openrtb/...  # vast can't pass go vet and golint right now
+GOFMT_FILES?=$$(find . -not -path "./vendor/*" -type f -name '*.go')
+PKG_PREFIX=github.com/Vungle/vungo
+PKGS=$(PKG_PREFIX)/openrtb $(PKG_PREFIX)/vast
+
+export GO111MODULE=on
+
+default: test
+
+# build
+build:
+	go build $(PKGS)
+
+# test runs the unit tests
+test:
+	go test $(TESTARGS) ./...
```

Comment:
where is TESTARGS defined?

Reply:
It's not defined by makefile itself, but can be customized by caller. If no env var, this will just be replaced by empty string.